modules/packages/ZMQ.chpl

```diff
@@ -419,6 +421,9 @@ module ZMQ {
   pragma "no doc"
   const unset = -42;
 
+  pragma "no doc"
+  const parallelAssignThreshold = 2*1024*1024;
```

Comment:
I think we should just use the existing `parallelAssignThreshold` defined in https://github.com/chapel-lang/chapel/pull/16418 instead of creating a duplicate that's not user-settable here.